api/lib/data/copy/check_in.yml

```diff
@@ -27,6 +25,16 @@ greeting:
         - you're a star ★️!
         - you're looking great today! 👀
 
+nps:
```

Comment:
Wording for these questions from https://legacy.hackclub.com/q1_q2_leader_survey.html